Assets/MRTK/Core/Interfaces/InputSystem/IMixedRealityFocusProvider.cs

```diff
@@ -89,6 +89,22 @@ public interface IMixedRealityFocusProvider : IMixedRealityService, IMixedRealit
         /// <typeparam name="T">The type of pointers to request. Use IMixedRealityPointer to access all pointers.</typeparam>
         IEnumerable<T> GetPointers<T>() where T : class, IMixedRealityPointer;
 
+        /// <summary>
+        /// Gets the first registered pointer that satisfies a specific condition.
+        /// </summary>
+        /// <typeparam name="T">The type of pointers to request. Use IMixedRealityPointer to access all pointers.</typeparam>
+        /// <param name="predicate">A function that returns true when the parameter satisfies a condition.</param>
+        /// <returns>The first pointer in the collection that satisfies the specified condition.</returns>
+        T GetFirstPointerWhere<T>(System.Predicate<T> predicate) where T : class, IMixedRealityPointer;
```

Comment:
I wonder if we can approach these as extension methods instead. Changing this interface would be a breaking change for anybody who has rolled their own focus provider, and I know of a handful of people who have.